universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java

```diff
@@ -110,26 +115,53 @@ private ContainerTag topContent(String titleText, String infoTextLine1, String i
 
   private ContainerTag mainContent(
       Messages messages,
-      ImmutableList<ProgramDefinition> programs,
+      ImmutableList<ProgramDefinition> draftPrograms,
+      ImmutableList<ProgramDefinition> activePrograms,
       long applicantId,
       Locale preferredLocale) {
     return div()
-        .withId("main-content")
-        .withClasses(Styles.MX_6, Styles.MY_4, StyleUtils.responsiveSmall(Styles.M_10))
-        .with(
-            h2().withText(messages.at(MessageKey.TITLE_PROGRAMS.getKeyName()))
-                .withClasses(Styles.BLOCK, Styles.MB_4, Styles.TEXT_LG, Styles.FONT_SEMIBOLD))
+        .withClasses(Styles.FLEX, Styles.JUSTIFY_CENTER)
         .with(
             div()
-                .withClasses(ApplicantStyles.PROGRAM_CARDS_CONTAINER)
+                .withId("main-content")
+                .withClasses(Styles.MX_AUTO, Styles.MY_4, StyleUtils.responsiveSmall(Styles.M_10))
+                .with(
+                    h2().withText(messages.at(MessageKey.TITLE_PROGRAMS.getKeyName()))
+                        .withClasses(
+                            Styles.BLOCK, Styles.MB_4, Styles.TEXT_XL, Styles.FONT_SEMIBOLD))
+                .with(
+                    h2().withText(messages.at(MessageKey.TITLE_PROGRAMS_IN_PROGRESS.getKeyName()))
```

Comment:
question (blocking): Does the applicant still see both of these `h2`s for in-progress and new programs even if there are no programs beneath them? If so, we should probably not do that and only show these subtitles when there are programs under both... Or at least only show the subtitle of the one that has any, if that makes sense? But I think it would be kind of weird on your first visit if you saw:

# Programs and services
## New programs
<program cards>

Likewise this is awkward:

#Programs and services
## In-progress programs
<no content>
## New programs
<all program cards>

Reply:
good catch!!! fixing now.